techeria_controller/templates/index.html

```diff
@@ -654,13 +654,87 @@ <h2 class="custom_heading text-center">
   <!-- end info_section -->
 
   <!-- footer section -->
+   <!-- footer section -->
   <section class="container-fluid footer_section">
-    <p>
-      © 2020 All Rights Reserved By
-      <a href="https://html.design/">Free Html Templates</a>
-    </p>
+    <div class="footer-top-area">
+        <div class="zigzag-bottom"></div>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3 col-sm-6">
+                    <div class="footer-about-us">
+                        <h2><span style="color: white">Techeria</span></h2>
+                        <h2><span style="color: white">Our Values</span></h2>
+                        <p>We are an e-commerce application, focused on making sure all technology products can be purchased in one space with the best deals so you can save your precious time without having to look through multiple websites.</p>
+                        <div class="footer-social">
+                            <a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+                            <a href="#" target="_blank"><i class="fa fa-twitter"></i></a>
+                            <a href="#" target="_blank"><i class="fa fa-youtube"></i></a>
+                            <a href="#" target="_blank"><i class="fa fa-linkedin"></i></a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-3 col-sm-6">
+                    <div class="footer-menu">
+                        <h2 class="footer-wid-title" style= "color: white">User Navigation </h2>
+                        <ul>
+                            <li><a href="#">My account</a></li>

```

Comment:
For each of these, we should make sure the letter to the start of each word is capitalized so that it is better for the user experience/UI.

Reply:
it's nice that you have added navigation. we should also add Terms of Use